tests/pytests/test_tags.py

```diff
@@ -160,3 +160,21 @@ def testIssue1305(env):
     res = env.cmd('ft.search', 'myIdx', '~@title:{wor} ~@title:{hell}', 'WITHSCORES')[1:]
     res = {res[i]:res[i + 1: i + 3] for i in range(0, len(res), 3)}
     env.assertEqual(res, expectedRes)
+
+def testTagCaseSensitive(env):
+    env.expect('FT.CREATE idx1 SCHEMA t TAG').ok()
+    env.expect('FT.CREATE idx2 SCHEMA t TAG CASESENSITIVE').ok()
+    env.expect('FT.CREATE idx3 SCHEMA t TAG SEPARATOR .').ok()
+    env.expect('FT.CREATE idx4 SCHEMA t TAG SEPARATOR . CASESENSITIVE').ok()
+    env.expect('FT.CREATE idx5 SCHEMA t TAG CASESENSITIVE SEPARATOR .').ok()
```

Comment:
Will this work on coordinator?

Reply:
@rafie It should work with the coordinator. For this sort of command, the coordinator does not do much.